File: benchmarks/c/01_fibonacci.c
// Fibonacci Benchmark - C Implementation
// Recursive Fibonacci calculation

#include <stdio.h>

long fib(int n) {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

int main() {
    int n = 35;
    long result = fib(n);
    printf("fib(%d) = %ld\n", n, result);
    return 0;
}

File: benchmarks/rust/01_fibonacci.rs
// Fibonacci Benchmark - Rust Implementation
// Recursive Fibonacci calculation

fn fib(n: i64) -> i64 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let n: i64 = 35;
    let result = fib(n);
    println!("fib({}) = {}", n, result);
}
